lua/typescript-tools/utils.lua

```diff
@@ -71,11 +71,16 @@ function M.is_nightly()
   return type(v) ~= "boolean" and v ~= nil or v
 end
 
+function M.get_clients(filter)
+  local get_clients = vim.version.ge(vim.version(), { 0, 10, 0 }) and vim.lsp.get_clients
```

Comment:
Is the `vim.version` check really needed? Given that `get_clients` and `get_active_clients` are identical, simply doing `local get_clients = vim.lsp.get_clients or vim.lsp.get_active_clients` achieves the same thing.